posthog/api/organization.py

```diff
@@ -148,21 +148,23 @@ class OrganizationViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     ordering = "-created_by"
 
     def dangerously_get_permissions(self):
-        permission_classes = self.permission_classes
-
-        # When listing there is no individual object to check for
         if self.action == "list":
-            permission_classes = [permissions.IsAuthenticated, APIScopePermission]
-        elif self.action == "create":
+            return [permission() for permission in [permissions.IsAuthenticated, APIScopePermission]]
```

Comment:
```suggestion
            return [permissions.IsAuthenticated(), APIScopePermission()]
```

Wouldn't that be shorter?

Reply:
_yes_ but I was aiming for consistency over brevity. Trying to stick to the other methods and how DRF does things